pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -174,14 +174,25 @@ message Mount {
     MountPropagation propagation = 5;
 }
 
+enum NamespaceMode {
+    POD       = 0;
+    CONTAINER = 1;
+    NODE      = 2;
+}
```

Comment:
These terms are a little bit different from `host_xxx` pattern we used before. So mind to add some comments for them? Like: what is `NamespaceMode=NODE` means? (Seems to be `host_network=true`)

Reply:
+1 for adding more comments.

Reply:
+1 of adding comments

maybe rename `NODE` to `HOST`? which I think is consistent with hostNetwork mode

Reply:
Yup, you're all right, the comments were insufficient. I've updated them now, please take another look.

@feiskyer I chose `NODE` because it's well defined by Kubernetes and I thought we should use the Kubernetes terminology where possible. It does create an inconsistency with v1.Pod (which itself uses host & node inconsistently) but I expect that a v2.Pod would replace the namespace booleans with an enum matching this one.